app/code/community/Bolt/Boltpay/Model/ShippingAndTax.php

```diff
@@ -195,10 +205,12 @@ public function getShippingAndTaxEstimate( Mage_Sales_Model_Quote $quote, $boltO
                 return $response;
             }
 
-            $this->applyShippingRate($quote, null);
+            $this->applyShippingRate($quote, null, false);
+            benchmark('Creating New Estimate: Cleared all applied shipping rates');
 
             $shippingAddress = $quote->getShippingAddress();
             $shippingAddress->setCollectShippingRates(true)->collectShippingRates()->save();
```

Comment:
@danac-gs 
Here is where we collect shipping rates on the addresses for the one and only time.